app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Invoices Index </h1>
+
+<% @invoices.each do |invoice| %>
+    <p><%= link_to invoice.id, admin_invoices_path("#{invoice.id}") %></p>
```

Comment:
Same comment Sierra made on the other PR. The string interpolation is unnecessary.